dpe/Cargo.toml

```diff
@@ -20,6 +20,7 @@ disable_csr = []
 disable_internal_info = []
 disable_internal_dice = []
 disable_retain_parent_context = []
+disable_export_cdi = []
```

Comment:
The specification has a separate ABI if `EXPORT_CDI` is set.

Should this compilation flag also change the shape of the `DeriveContextResp` type?